presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestTpchQueries.java

```diff
@@ -66,4 +66,14 @@ public void testTpchTinyTables()
         assertQuery(session,
                 "SELECT l_linenumber, l_orderkey, l_discount FROM lineitem WHERE l_discount > 0.2");
     }
+
+    @Test
+    public void testTpchDateFilter() {
+        Session session = Session.builder(getSession())
```

Comment:
This code repeats. Consider extracting a helper method, possible taking catalog name as a parameter.